portfolio/src/main/webapp/assets/js/main.js

```diff
@@ -62,9 +62,34 @@ jQuery(document).ready(function($) {
         }
     );
     
-    
     /* Github Activity Feed - https://github.com/caseyscarborough/github-activity */
     GitHubActivity.feed({ username: "jackdavidweber", selector: "#ghfeed" });
 
+});
+
+
+async function getTestimonials() {
+  const response = await fetch('/data');
+  const result = await response.json();
+  
+  arrTstls = result["arrTestimonials"];
+
+  const tstlsListElement = document.getElementById('testimonials');
+  tstlsListElement.innerHTML = '';
+
+  for (let i = 0; i < arrTstls.length; i++) {
+      tstlsListElement.appendChild(createListElement(arrTstls[i]));
+  }
+
+  const numTstlsElement = document.getElementById('numTestimonials');
+  numTstlsElement.innerHTML = "<p>Number of Testimonials Visible: "+ arrTstls.length.toString() + "</p>"
+
+
```

Comment:
Remove extra new lines at the bottom of the function.

Reply:
got it!